pkg/tuple/ipv4.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
```

Comment:
Please add `pkg/tuple` to be owned by `cilium/bpf` in CODEOWNERS.

Reply:
Ok, done.